server/core/config.go

```diff
@@ -17,18 +17,29 @@
 package core
 
 import (
+	"os"
+	"runtime"
+	"time"
+
 	"github.com/apache/servicecomb-service-center/pkg/log"
 	"github.com/apache/servicecomb-service-center/pkg/plugin"
 	"github.com/apache/servicecomb-service-center/pkg/util"
 	pb "github.com/apache/servicecomb-service-center/server/core/proto"
 	"github.com/apache/servicecomb-service-center/version"
 	"github.com/astaxie/beego"
-	"os"
-	"runtime"
 )
 
 const (
 	INIT_VERSION = "0"
+
+	defaultServiceClearInterval = 12 * time.Hour //0.5 day
+	defaultServiceClearTime     = 24 * time.Hour //1 day
+
+	minServiceClearInterval = 1 * time.Second
+	minServiceClearTime     = 1 * time.Second
```

Comment:
every second clear a services is impossibel senario in real world 

make it to 30s